src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -57,7 +53,7 @@ public async Task<ActionResult> Index(string hashedAccountId, string reservation
             if (FeatureToggles.Features.HomePage.Enabled || !HasPayeScheme(response.Data) && !HasOrganisation(response.Data))
             {
                 var unhashedAccountId = _hashingService.DecodeValue(hashedAccountId);
-                response.Data.AccountViewModel = await _portalClient.GetAccount(unhashedAccountId);
+                response.Data.AccountViewModel = await _portalClient.GetAccount(unhashedAccountId, false);
```

Comment:
As per [CON-663 comment](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1740/files#r302444297):

The signature of this method has changed in the latest version of the portal client, so this branch needs updating with the latest version of the package.
Also, the hasPayeScheme parameter shouldn't be hard-coded to false.
This PR should really include the CON-668-vacancy-multiple-vacancies-account-web-fe branch, where that work has already been done.